### DISCUSSION TITLE
Is the custom f0 method even necessary?

### DISCUSSION THREAD

**Discussion by Meldoner** [voicepaw/so-vits-svc-fork#360] [Q&A] [answered]
I have seen many people make models just by running pre hubert without the -fm command

**Comment by Mangio621** [comment, marked as answer]
Currently I'm using crepe on pre hubert cause apparently its a lot more accurate with pitch and has less failure cases. I've produced "better" results i think with crepe on pre hubert instead of dio or any other.
